lerna.json

```diff
@@ -1,11 +1,12 @@
 {
-  "lerna": "3.11.0",
+  "lerna": "3.19.0",
   "packages": [
+    "packages/core",
+    "packages/modeling",
     "packages/cli",
     "packages/desktop",
     "packages/web",
     "packages/examples",
-    "packages/core",
     "packages/io/*",
     "packages/utils/*"
```

Comment:
This should also have "packages/ui/*" added.

Reply:
no, they are not included for a good reason: some of the dependencies in there are NOT compatible with CI (travis or others), and cannot be included this way.
I spent ages finding workarounds, but nothing worked so far.
We can revisit later.